cmd/talosctl/cmd/talos/config.go

```diff
@@ -186,6 +188,106 @@ var configAddCmd = &cobra.Command{
 	},
 }
 
+// configRemoveCmdFlags represents the `config remove` command flags.
+var configRemoveCmdFlags struct {
+	noconfirm bool
+	dry       bool
+}
+
+// configRemoveCmd represents the `config remove` command.
+var configRemoveCmd = &cobra.Command{
+	Use:   "remove <context>",
+	Short: "Remove a context",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		pattern := args[0]
+		if pattern == "" {
```

Comment:
Unncessary check. `cobra.ExactArgs(1)` will make sure an argument is passed.

Reply:
Not exactly. You can still do `""`.